### PR TITLE
Add 2023 tax bills

### PR DESCRIPTION
**Pull request by erhla**

### CODE REVIEW

**Comment by erhla** [comment]
Oops. Here's the 2023 tax bills to add to your branch.

**Review by dfsnow — COMMENTED** [review]
@erhla Can you fix the linting errors on `sample_tax_bills_detail.R`?

**Comment by dfsnow** [comment]
@erhla There's also the `data-raw/sample_tax_bills/sample_tax_bills_summary.csv` which needs to be updated manually (by manually looking at the exemption details on each bill). Lmk if you wanna do it, otherwise we can merge this and I'll grab it later.

File: data-raw/sample_tax_bills/sample_tax_bills_summary.R [hunk 1]
```diff
@@ -3,9 +3,28 @@ library(stringr)
...
+  distinct(year, pin, class) %>%
+  mutate(class = as.character(class)) %>%
+  filter(year == max(year))
+
+readr::read_csv(
+  sum_path,
+  col_types = cols(pin = "c", tax_code = "c", class = "c")
+) %>%
+  bind_rows(key_cols) %>%
+  write_csv(sum_path)
```
Comment (dfsnow) [inline data-raw/sample_tax_bills/sample_tax_bills_summary.R:22]: **issue (blocking):** I get the intent but let's not add this programmatically. It's fine if you wanna use it to help jumpstart the manual work on each bill though.

Also, somehow it's mangling the CSV, see below.

File: data-raw/sample_tax_bills/sample_tax_bills_summary.csv
Comment (dfsnow) [inline data-raw/sample_tax_bills/sample_tax_bills_summary.csv:1]: **issue (blocking):** Looks like some of the 0000 PINs get mangled and converted to scientific notation here. No bueno.

**Review by dfsnow — CHANGES_REQUESTED** [review]
@erhla

**Pull request merged**